example_project/example_project/orchestra_settings.py

```diff
@@ -108,7 +108,10 @@ def setup_orchestra(settings_module_name):
 
     # Don't authenticate users without a view explicitly calling for it
     settings.REST_FRAMEWORK = {
-        'DEFAULT_AUTHENTICATION_CLASSES': (),
+        'DEFAULT_AUTHENTICATION_CLASSES': (
```

Comment:
reminding

I can see us removing BasicAuth - we don't use it and probably shouldn't allow it